src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -47,7 +54,7 @@ module Log =
         | None -> f log
         | Some retryPolicy ->
             let withLoggingContextWrapping count =
-                let log = if count = 1 then log else log |> prop contextLabel count
+                let log = if count = 1 then log else Tracing.addTag("eqx.attempts", count); log |> prop contextLabel count
```

Comment:
I've made it so we have 1 span for the all the attempts rather than 1 per.

Npgsql has tracing built-in as well so you can see the individual queries made to the DB as well.

I'd prefer putting it on all spans so I could do queries like `show p99 duration by attempt count`, but you've voiced a preference for not including the default value of 1

Reply:
Have I made it like that? 🤦 I don't think so

Reply:
if its cumbersome to query when irregular, `attempt 1` is not a problem to have redundantly in there

Reply:
Note there is an outer `attempt` wrt the OCC retry loop, see https://github.com/jet/equinox/blob/master/src/Equinox.Core/Category.fs#L43
if that needs to be passed inwards as an accumulating context, that can be done... (or does there need to be an outer activity?)

Reply:
Oh interesting, so we have `n*m` retries!

Reply:
I'd say in the vast majority of cases the OCC retry (with reload) is far more important of course.
But when the read/write retry is needed, it's really needed.
Thanks to Async CTs you can 'simply' enforce a timeout when it all goes horribly wrong ;)